src/main/java/com/mixpanel/android/viewcrawler/ViewSnapshot.java

```diff
@@ -114,6 +115,8 @@ public void snapshots(UIThreadSet<Activity> liveActivities, OutputStream out) th
                 Log.e(LOGTAG, "Exception thrown during screenshot attempt", e);
             }
         }
+        writer.write("\"\"");
+        writer.flush();
```

Comment:
This makes me nervous. What exception are we throwing? (I'd also prefer if the control flow was to emit the quotes in an exception handler than having the return inside of the try block.